src/main/java/org/sciborgs1155/robot/intake/Intake.java

```diff
@@ -0,0 +1,38 @@
+package org.sciborgs1155.robot.intake;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.wpilibj2.command.button.Trigger;
+import monologue.Logged;
+import org.sciborgs1155.robot.intake.IntakeIO.NoIntake;
+import org.sciborgs1155.robot.intake.IntakeIO.RealIntake;
+
+public class Intake extends SubsystemBase implements Logged, AutoCloseable {
+  private final IntakeIO intake;
+
+  public Intake(IntakeIO intake) {
+    this.intake = intake;
+    setDefaultCommand(run(() -> intake.setPower(0)));
+  }
+
+  public Command intake() {
+    return run(() -> intake.setPower(IntakeConstants.INTAKE_SPEED));
+  }
+
+  public Command outtake() {
+    return run(() -> intake.setPower(-IntakeConstants.INTAKE_SPEED));
+  }
+
+  public Trigger beambreakTrigger() {
+    return new Trigger(intake::getBeambreakValue);
+  }
+
+  public static Intake create(boolean isReal) {
```

Comment:
move this method to the top, and use `Robot.isReal()` instead
also, add a `createNone` method that creates a NoIntake